e2e/features/pa-appeal-pay-now.feature

```diff
@@ -0,0 +1,145 @@
+    And I click the `Continue` button
+
+    Then I am on the `Your client's address` page
+    When I select `Yes` for the `Does the appellant have a fixed address?` field
+    And I see the text `Enter a UK postcode`
+    And I type `SW1A 2AA` for the `Enter a UK postcode` field
+    And I click the `Find address` button
+    And I see the text `Select an address`
+    And I wait for any found addresses to load
+    And I select `Prime Minister & First Lord Of The Treasury, 10 Downing Street, London` for the `Select an address` field
+    And I see the text `Building and Street`
+    And I click the `Continue` button
+
+    Then I am on the `The appellant's contact preference` page
+    And I select `Text message` for the `Communication Preference` field
+    And I type `07930111111` for the `Mobile phone number` field
+    And I click the `Continue` button
+
+    Then I am on the `Type of appeal` page
+    And I select `Refusal of protection claim` for the `Type of appeal` field
+    And I click the `Continue` button
+
+    Given I am on the `The grounds of your appeal` page
+    And I should see the text `Removing the appellant from the UK would breach the UK's obligation under the Refugee Convention`
+    And the `Continue` button is disabled
+
+    When I click the `Removing the appellant from the UK would breach the UK's obligation under the Refugee Convention` label
+    Then the `Continue` button is enabled
+
+    When I click the `Removing the appellant from the UK would breach the UK's obligation under the Refugee Convention` label
+    And I click the `Removing the appellant from the UK would breach the UK's obligation in relation to persons eligible for a grant of humanitarian protection` label
+    Then the `Continue` button is still enabled
+
+    When I click the `Removing the appellant from the UK would breach the UK's obligation under the Refugee Convention` label
+    And I click the `Removing the appellant from the UK would be unlawful under section 6 of the Human Rights Act 1998` label
+    And I click the `Continue` button
+
+    And I complete the `New matters` page
+    And I complete the `Has your client appealed against any other UK immigration decisions?` page
+    And I complete the `Legal representative details` page
+
+    Then I select `Decision without a hearing. The fee for this type of appeal is £80` for the `How do you want the appeal to be decided?` field
+    And I select `Decision with a hearing. The fee for this type of appeal is £140` for the `How do you want the appeal to be decided?` field
+    And I click the `Continue` button
+
+    Then I should see the text `Don't have Payment by Account?`
+    And I should see the `Apply to use Payment by Account` link
+
+    And I click the `Pay after submitting the appeal by card` label
+    And I click the `Pay after submitting the appeal using Payment by Account` label
```

Comment:
Same as above. Any reason we are also clicking this option - seems unnecessary?

Reply:
Additional click to check the selection works.